tools-v2/pkg/cli/command/curvebs/stop/volumeSnapshot/stop_volume.go

```diff
@@ -0,0 +1,171 @@
+package volumeSnapshot
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+)
+
+const (
+	version = "0.0.6"
+	limit   = "100"
+	offset  = "0"
+)
+
+type Stop struct {
+	serverAddress []string
+	timeout       time.Duration
+
+	User     string
+	FileName string
+	UUID     string
+
+	messages []string
+}
+
+func newStopSnapShot(serverAddress []string, timeout time.Duration, user, fileName, uuid string) *Stop {
+	return &Stop{
+		serverAddress: serverAddress,
+		timeout:       timeout,
+		User:          user,
+		FileName:      fileName,
+		UUID:          uuid,
+	}
+}
+
+type Record struct {
+	UUID string `json:"UUID"`
+	User string `json:"User"`
+	File string `json:"File"`
+
+	Result string `json:"Result"`
+}
+
+func (s *Stop) queryStopBy() []*Record {
+	records := s.getSnapShotListAll()
+	return records
+}
+
+func (s *Stop) getSnapShotListAll() []*Record {
+	var receiveRecords []*Record
+	params := QueryParams{
+		Action:  "GetFileSnapshotList",
+		Version: version,
+		User:    s.User,
+		UUID:    s.UUID,
+		File:    s.FileName,
+		Limit:   limit,
+		Offset:  offset,
+	}
+	for {
+		records := s.getSnapShotList(params)
+		if len(records) == 0 || records == nil {
+			break
+		}
+		receiveRecords = append(receiveRecords, records...)
+
+		params.Offset = params.Offset + params.Limit
+
+	}
+	return receiveRecords
+}
+
+type QueryParams struct {
+	Action  string `json:"Action"`
+	Version string `json:"Version"`
+	User    string `json:"User"`
+	File    string `json:"File"`
+	UUID    string `json:"UUID"`
+	Limit   string `json:"limit"`
+	Offset  string `json:"Offset"`
+}
+
+func (s *Stop) getSnapShotList(params QueryParams) []*Record {
+	var resp struct {
+		Code       string
+		Message    string
+		RequestId  string
+		TotalCount int
+		SnapShots  []*Record
+	}
+	err := s.query(params, &resp)
+	if err != nil || resp.Code != "0" {
+		fmt.Printf("get snapshot list fail, error=%s\n", err)
```

Comment:
Try not to output errors in the middle, but return an error, cmdError, etc.
You can refer to the implementation of other commands